requirements.txt

```diff
@@ -39,7 +39,7 @@ django-cors-headers==3.10.0
     # via -r requirements.in
 django-environ==0.7.0
     # via -r requirements.in
-django-helusers==0.8.1
+django-helusers==0.9.0
```

Comment:
Is it now possible to remove the `drf-oidc-auth` dependency?

Reply:
It's required by helsinki-profile-gdpr-api.

Reply:
But can be removed from `requirements.in`?

Reply:
True that.